app/controllers/fhir/r4/api_controller.rb

```diff
@@ -766,9 +781,43 @@ def status_not_found
     end
   end
 
-  def status_unprocessable_entity(errors = [])
+  # 422 response with specific validation errors.
+  def status_unprocessable_entity(resource, fhir_map, req_json)
+    outcome = FHIR::OperationOutcome.new(issue: [])
+
+    resource&.errors&.messages&.each do |attribute, errors|
+      next unless VALIDATION.key?(attribute) || attribute == :base
+
+      fhir_path = fhir_map&.dig(attribute, :path)
+
+      # Extract the original value from the request body using FHIRPath
+      if fhir_path&.present?
+        begin
+          value = nil
+          Rails.logger.silence do
+            value = FHIRPath.evaluate(fhir_path, req_json)
+          end
```

Comment:
Is this because we've redirected the logging to the rails logger, but we also don't want to log it via the rails logger?

Reply:
That's right. As far as I could tell there was no way to suppress the logging to STDOUT.